openmetadata-airflow-apis/openmetadata_managed_apis/workflows/ingestion/common.py

```diff
@@ -255,7 +269,8 @@ def build_dag_configs(ingestion_pipeline: IngestionPipeline) -> dict:
         "schedule_interval": ingestion_pipeline.airflowConfig.scheduleInterval,
         "tags": [
             "OpenMetadata",
-            ingestion_pipeline.displayName or ingestion_pipeline.name.__root__,
+            clean_name_tag(ingestion_pipeline.displayName)
+            or clean_name_tag(ingestion_pipeline.name.__root__),
             ingestion_pipeline.pipelineType.value,
             ingestion_pipeline.service.name,
```

Comment:
should we also modify service name to 100 chars?

Reply:
fair enough, hoped noone would name it that long but since we don't have a max. Good call